rc/lsp.kak

```diff
@@ -345,10 +346,11 @@
 declare-option -hidden line-specs lsp_code_lenses 0 '0| '
 declare-option -hidden str lsp_project_root
 
+declare-option -hidden str lsp_modeline_breadcrumbs ""
 declare-option -hidden str lsp_modeline_code_actions
 declare-option -hidden str lsp_modeline_progress ""
 declare-option -hidden str lsp_modeline_message_requests ""
-declare-option -hidden str lsp_modeline '%opt{lsp_modeline_code_actions}%opt{lsp_modeline_progress}%opt{lsp_modeline_message_requests}'
+declare-option -hidden str lsp_modeline '%opt{lsp_modeline_breadcrumbs}%opt{lsp_modeline_code_actions}%opt{lsp_modeline_progress}%opt{lsp_modeline_message_requests}'
```

Comment:
fixed the system tests (hope it's not flaky)

The system tests are woefully undocumented but they are fairly simple to use.

Each system test is an executable shell script.
When run by itself it does not check the output.
Since it doesn't need to capture any output it can take control of the TTY.
This means you can insert a `test_tmux attach` line to attach to a running test and debug interactively.

For running the test with checks use something like `sh -x test/run test/path-to-test.sh`.
Probably we can improve the interface.